modules/core/src/main/scala/lucuma/odb/api/repo/OneToManyUnique.scala

```diff
@@ -49,17 +49,15 @@ object OneToManyUnique {
     bToA:  SortedMap[B, A]
   ): OneToManyUnique[A, B] = new OneToManyUnique[A, B] {
 
+    // "Overwrites" existing links - there is no notification for the elements that are unlinked
```

Comment:
This matches the current behavior for observation <-> pointing and doesn't allow for sending edit events for the constraint sets that are silently "unlinked". This will also be fixed in the rewrite.